static/app/components/replays/breadcrumbs/replayComparisonModal.tsx

```diff
@@ -53,20 +56,33 @@ export default function ReplayComparisonModal({
         </ModalHeader>
       </Header>
       <Body>
-        <StyledParagraph>
-          {tct(
-            'This modal helps with debugging hydration errors by diffing the dom before and after the app hydrated. [boldBefore:Before Hydration] refers to the html rendered on the server. [boldAfter:After Hydration] refers to the html rendered on the client. This feature is actively being developed; please share any questions or feedback to the discussion linked above.',
-            {
-              boldBefore: <strong />,
-              boldAfter: <strong />,
-            }
+        <Grid>
+          <StyledParagraph>
+            {tct(
+              'This modal helps with debugging hydration errors by diffing the dom before and after the app hydrated. [boldBefore:Before Hydration] refers to the html rendered on the server. [boldAfter:After Hydration] refers to the html rendered on the client. This feature is actively being developed; please share any questions or feedback to the discussion linked above.',
+              {
+                boldBefore: <strong />,
+                boldAfter: <strong />,
+              }
+            )}
+          </StyledParagraph>
+
+          {isSameTimestamp ? (
+            <Alert type="warning" showIcon>
+              {t(
+                "Sentry wasn't able to identify the correct event to display a diff for this hydration error."
```

Comment:
```suggestion
                "Sentry wasn't able to identify the correct event for which to display a diff for this hydration error."
```

Reply:
something about this sentence reads weird, trying to think of a better way to phrase it